styleguide-app/Examples/AssignmentIcon.elm

```diff
@@ -53,5 +53,20 @@ example =
                 [ ( "startPrimary", AssignmentIcon.startPrimary )
                 , ( "startSecondary", AssignmentIcon.startSecondary )
                 ]
+            , IconExamples.view "Activities"
+                [ ( "assessment", AssignmentIcon.assessment )
+                , ( "practice2", AssignmentIcon.practice2 )
+                , ( "standards", AssignmentIcon.standards )
+                , ( "writing", AssignmentIcon.writing )
+                ]
+            , IconExamples.view "Practice & Assessment"
+                [ ( "assignArrow", AssignmentIcon.assignArrow )
+                , ( "home", AssignmentIcon.home )
+                , ( "passageQuiz", AssignmentIcon.passageQuiz )
+                , ( "planningDiagnostic", AssignmentIcon.planningDiagnostic )
+                , ( "quiz2", AssignmentIcon.quiz2 )
+                , ( "unitDiagnostic", AssignmentIcon.unitDiagnostic )
+                , ( "allLibrary", AssignmentIcon.allLibrary )
```

Comment:
What does allLibrary mean?

Reply:
Looking at it in the UI, it's book spines. 🤔 I wonder if this icon should be named after what it looks like, rather than what it means in the new UI.